config.py.sample

```diff
@@ -3,4 +3,6 @@ prefix = ['']  # List of strings. Defaults to '!' and '.' if left empty
 secret = ""  # Website secret as string
 is_beta = False  # Used to determine if bot is beta or production
 github_username = ""  # Github username as string
-github_password = ""  # Github password as string
\ No newline at end of file
+github_password = ""  # Github password as string
+warn_db_storage = False # Used to determine if we should use a database for storing warns or not.
```

Comment:
Again, please adjust these comments to format the same as the other ones. Also, two spaces before the `#`, not one. 
`# Bool for if MongoDB database is in use`
`# MongoDB database address as string`

Reply:
I'll do the last adjustment for pep8 stuff.